### PR TITLE
Peer probe and volume test case working

### PR DESCRIPTION
**Pull request by nishith-vihar**
Peer probe and volume test cases are working
successfully integrating with all the components.

Updates: #127
Signed-off-by: Nishith Vihar Sakinala <nsakinal@redhat.com>

### CODE REVIEW

File: tests/functional/glusterd/test_volume_create_delete.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class TestCase(ParentTest):
     for volume creation and deletion
     """
 
-    def volume_create_delete_test(self):
+    def run_test(self):
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_volume_create_delete.py:16]: Why the change in function name?
Moreover, it is same as the previous test case

Comment (nishith-vihar) [reply tests/functional/glusterd/test_volume_create_delete.py:16]: the runner_thread called the run_test function which is in parent test. So here we are overriding the run_test function from parent test so that the runner thread call the run_test function in the test case.

Comment (nishith-vihar) [reply tests/functional/glusterd/test_volume_create_delete.py:16]: As of now all the test cases will have run_test  as the function name

File: tests/functional/glusterd/test_peer_probe_detach.py [hunk 2]
```diff
@@ -22,16 +22,18 @@ def test_peer_probe_detach(self):
...
+            self.redant.pool_list(server1)
 
-            self.redant.peer_detach("192.168.122.220", "192.168.122.161")
+            self.redant.peer_detach(server1, server2)
 
-            self.redant.glusterd_stop("192.168.122.220")
+            self.redant.glusterd_stop(server1)
 
         except Exception as e:
             print(f"Test is failed:{e}")
```
Comment (neesingh-rh) [inline tests/functional/glusterd/test_peer_probe_detach.py:39]: I think we can now handle assignment of TEST_RES with Fail also in exception as parent class is now inherited.

**Review by srijan-sivakumar — APPROVED** [review]

**Review by neesingh-rh — APPROVED** [review]
